cmd/auth/login.go

```diff
@@ -57,6 +57,12 @@ var loginCmd = &cobra.Command{
 		}
 
 		if configureCluster {
+			// We need to save profile before it's used to initialise new workspace client.
+			// Otherwise it will complain about non existing profile
+			err = databrickscfg.SaveToProfile(ctx, &cfg)
+			if err != nil {
+				return err
+			}
```

Comment:
The side effect of saving the databrickscfg file is that it saves a backup file. By saving twice it would overwrite that backup file on the second save. Instead, I suspect that if we clear the `Profile` key in the config on L56, then the line below here will work out of the box, and the changes to the loader + ops are not necessary.

Reply:
Indeed, not providing Profile in this case will help